semantic/src/site/globals/site.overrides

```diff
@@ -374,27 +393,6 @@ a:hover {
   font-weight: bold;
 }
 
-.sup .ui.tiny.progress {
-  margin-bottom: 4px !important;
-}
-
-.sup .ui.form .required.fields:not(.grouped) > .field > label:after {
-  color: rgb(248, 248, 248) !important;
-}
-
-/* Table Striping */
-.stripes .ui.grid:nth-child(even) {
-  background: rgb(248, 248, 248);
-}
-
-.stripes .ui.grid:nth-child(even) .stripedlabel {
-  background: white;
-}
-
-.stripes .ui.grid:nth-child(odd) .stripedlabel {
-  background: rgb(248, 248, 248);
-}
-
```

Comment:
Great, thanks for deleting this. I've been wanting to but wasn't sure it wasn't needed.